terraform/.gitignore

```diff
@@ -0,0 +1,34 @@
+# Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
```

Comment:
nice!

but this pr includes a .tfstate_back file

Reply:
yeah I renamed that when moving state to terraform cloud. we should be good now Thanks